examples/data_pipeline/benchmark_pytext_vocab.py

```diff
@@ -0,0 +1,185 @@
+from collections import (Counter, OrderedDict)
+import time
+from typing import List
+
+import torch
+from torchtext.experimental.datasets import AG_NEWS
+
+from pytext_vocab import ScriptVocab as ExperimentalScriptVocabulary
+from pytext.torchscript.vocab import ScriptVocabulary as PytextScriptVocabulary
+from pytext.data.utils import Vocabulary as PytextVocabulary
+
+
+def _run_benchmark_lookup(tokens, vocab, num_iters=1):
```

Comment:
Can we consolidate those `_run_benchmark_lookup*` funcs?

Reply:
I did my best to consolidate some of these functions. I still need to keep separate functions for the JITd for loop. Lmk what you think